external/ExternalIntegrationExample/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(gdb) Attach - ExternalIntegrationExample",
+            "type": "cppdbg",
```

Comment:
lldb ??

Reply:
I can try that.  I just took whatever vscode popped up as the default.